en/getting-started/first-game/index.md

```diff
@@ -362,165 +370,190 @@ export class GameManager extends Component {
 }
 ```
 
-Assign the Cube prefab that made previously to the `CubePrfb` property in GameManager Inspector.
+After saving the script and returning to the editor, you can see the new `CubePrfb` and `RoadLength` properties in the `GameManager` script component of the `GameManager` node. Then drag and drop the Cube's prefab resource from the __Assets__ panel into the `CubePrfb` property box.
 
 ![assign cube prefab](./images/assign-cube-prefab.png)
 
-The length of the road can be changed by modifying the value of `roadLength` in the __Properties__ panel for the `GameManager`.
+The `RoadLength` property is used to set the length of the runway. At this point, click the __Preview__ button to see the automatically generated runway:
 
-When previewing, the road is now automatically generated, however, because the __Camera__ does not follow the `Player`, the road behind cannot be seen. Changing the __Camera__ in the __Scene__ to be a child node of the `Player` can help solve this.
+![runway](./images/runway.gif)
+
+However, since the `Camera` does not follow the `Player`, we cannot see the runway afterwards, so we can set the `Main Camera` node as a child of the `Player` node:
```

Comment:
```suggestion
However, since the `Camera` does not follow the `Player`, the runway isn't visible afterward, set the `Main Camera` node as a child of the `Player` node:
```